docker-for-aws/index.md

```diff
@@ -38,6 +38,67 @@ using CloudFormation. For more about stable and beta channels, see the
   </tr>
 </table>
 
+## Deployment options
+
+There are two ways to deploy Docker for AWS, you can install inside of a pre-existing VPC, or you can have Docker for AWS create the VPC and everything that it needs. We recommend the second option, since it allows us to setup the environment the way in the best way possible. If you have to install in an existing VPC, that is fine, but it will require a little more work.
+
+### Create a new VPC
+This approach will create a new VPC, subnets, gateways and everything else that is needed in order to run Docker for AWS. It is the easiest way to get started, and requires the least amount of work.
+
+All you need to do it run the CloudFormation template, answer some questions, and you are good to go.
+
+### Install into an Existing VPC
```

Comment:
into -> with.

(am not sure if it is a good comment, but `into` sounds boxed and am not sure if that is the intended architecture as visualized by the AWS users)